browser/resources/settings/brave_overrides/page_visibility.js

```diff
@@ -48,6 +48,7 @@ function getPageVisibility () {
     privacy: alwaysTrueProxy,
     // custom properties
     braveSync: !loadTimeData.getBoolean('isSyncDisabled'),
+    braveWallet: loadTimeData.getBoolean('isBraveRewardsSupported'),
```

Comment:
Using `isBraveWalletAllowed` instead of `isBraveRewardsSupported`?

Reply:
🙀 Thanks for catching! Copy/paste strikes again 😱 Fixed in e13f9a459d02cfef658c478b215ccf8bbf9eae8e